test/Orleans.Transactions.Azure.Test/GoldenPathTransactionManagerTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Orleans.Runtime.Configuration;
+using Orleans.Transactions.Abstractions;
+using Orleans.Transactions.Tests;
+using TestExtensions;
+
+namespace Orleans.Transactions.Azure.Tests
+{
+    [TestCategory("Azure"), TestCategory("Transactions"), TestCategory("Functional")]
+    public class GoldenPathTransactionManagerTest : GoldenPathTransactionManagerTestRunner
+    {
+        private static readonly TimeSpan LogMaintenanceInterval = TimeSpan.FromMilliseconds(10);
+        private static readonly TimeSpan StorageDelay = TimeSpan.FromSeconds(30);
+
+        public GoldenPathTransactionManagerTest(ITestOutputHelper output)
+            : base(MakeTransactionManager(), LogMaintenanceInterval, StorageDelay, output)
+        {
+        }
+
+        private static ITransactionManager MakeTransactionManager()
+        {
+            ILoggerFactory loggerFactory = new LoggerFactory();
```

Comment:
You can use NullLoggerFactory.Instance if you want want your logger to log anything